src/adapters/MeetingsJSONAdapter.js

```diff
@@ -35,7 +60,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
    * @memberof MeetingsJSONAdapter
    */
   getMeeting(ID) {
-    return Observable.create((observer) => {
+    const json$ = Observable.create((observer) => {
```

Comment:
can we change this to `getMeeting$` to have a consistent naming pattern with the SDK for now?